lib/src/commands.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:bosun/bosun.dart';
+
+class HelpCmd extends Command {
+  final Command context;
+
+  HelpCmd(this.context)
+      : super(
+            command: 'help',
+            description: 'Displays helpful information about the current project',
+            subcommands: []);
+
+  @override
+  void run(List<String> args, Map<String, dynamic> flags) {
+    String commands = '';
+    for (var element in [...context.subcommands!, ...BosunCmd(context).subcommands!]) {
+      commands += '  ${element.command}\t${element.description ?? ''}\n';
+    }
+    print('''
+A command-line utility for Dart CLI development.
```

Comment:
Consumers of Bosun should be able to use it without anyone caring that's using Bosun under the hood. Perhaps we add a description on the root Bosun Command and populate this from that field.

Reply:
Now uses the `description` field of the root command.